hack/Makefile

```diff
@@ -14,7 +14,6 @@ generate-python-client:
 	./hack/generate.sh generate_python_client
 
 generate-mocks:
-	find . -name 'mock_*.go' -type f -delete
```

Comment:
but it means we're left with redundant mock files whenever we're deleting the mocked modules, no?

Reply:
If not removed, then `skipper make unit-test` fails with
```
vendor/github.com/ovirt/go-ovirt-client/mock.go:27:48: undefined: diskWithData
vendor/github.com/ovirt/go-ovirt-client/mock.go:34:48: undefined: datacenterWithClusters
vendor/github.com/ovirt/go-ovirt-client/newmock.go:76:18: undefined: datacenterWithClusters
vendor/github.com/ovirt/go-ovirt-client/newmock.go:131:42: undefined: datacenterWithClusters
vendor/github.com/ovirt/go-ovirt-client/newmock.go:139:52: undefined: datacenterWithClusters
```
because when the above line is in, then `skipper make generate` deletes the mock files
```
deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_cluster_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_cluster_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_datacenter.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_datacenter_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_datacenter_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_datacenter_list_clusters.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_attachment_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_attachment_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_attachment_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_attachment_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_downloadimage.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_get_internal.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_list_by_alias.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_update.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_uploadimage.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_disk_util.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_host_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_host_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_network_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_network_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_nic_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_nic_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_nic_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_nic_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_nic_update.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_storagedomain_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_storagedomain_get_disk.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_storagedomain_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_storagedomain_remove_disk.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_tag_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_tag_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_tag_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_tag_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_copy_disk.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_get_blank.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_template_wait_for_status.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_add_tag.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_optimize_pinning.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_remove.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_search.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_shutdown.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_start.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_stop.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_update.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vm_wait_for_status.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vnicprofile_create.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vnicprofile_get.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vnicprofile_list.go
        deleted:    vendor/github.com/ovirt/go-ovirt-client/mock_vnicprofile_remove.go
```

Reply:
ahhh...
so you can change to ``find . -name 'mock_*.go' -type f -not -path "vendor/*" -delete``

Reply:
+1 to Osher

Reply:
I'm afraid it must be exactly ``find . -name 'mock_*.go' -type f -not -path "vendor/*" -delete``. With latest changes no mock files are generated

Reply:
Nit: `find . -name 'mock_*.go' -type f -not -path './vendor/*' -delete`